lte/gateway/deploy/roles/magma/files/systemd/magma_envoy_controller.service

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[Unit]
+Description=Magma envoy controller service
+PartOf=magma@pipelined.service
+Before=magma@pipelined.service
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/environment
+ExecStartPre=/bin/bash /usr/local/bin/configure_envoy_namespace.sh setup
```

Comment:
@pshelar does this look fine to you? Also check the `magma_envoy.service` exec

Reply:
looks good.